frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import {
+  obcCount,
+  efficiencyValue,
+  savingsValue,
+  healthOfMCG,
+  resourceProviders,
+  noobaaCount,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: testName,
+    name: 'test-bucket',
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Tests at least one noobaa bucket is present', async () => {
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Creates an Object Bucket Claim and tests equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -n ${testName} -f -`);
+    await browser.sleep(10000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -n ${testName} -f -`);
+  });
+});
+
+describe('Tests Status Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks if Multi Cloud Gateway is in a healthy state', async () => {
+    await browser.wait(until.visibilityOf(healthOfMCG));
+    expect(healthOfMCG.isPresent()).toBe(true);
+  });
+});
+
+describe('Tests Object Data Reduction Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
+  });
+
+  it('Checks for savings value to be in acceptable data range', async () => {
```

Comment:
Again not a valid test. As the saving can be 0 for the initial thing and it's not necessary that savings always happens and depends on data that's present.

Reply:
So we accept no savings too.

Reply:
The point is savings should have some value we don't accept unavailable.